Spigot-Server-Patches/0407-offset-item-frame-ticking.patch

```diff
@@ -0,0 +1,27 @@
+From 076621f7e8b6ca21d6682dd53f006b3c283b8279 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
+Date: Tue, 30 Jul 2019 03:17:16 +0500
+Subject: [PATCH] offset item frame ticking
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityHanging.java b/src/main/java/net/minecraft/server/EntityHanging.java
+index 3b282a18a..f548e6a05 100644
+--- a/src/main/java/net/minecraft/server/EntityHanging.java
++++ b/src/main/java/net/minecraft/server/EntityHanging.java
+@@ -22,11 +22,13 @@ public abstract class EntityHanging extends Entity {
+     protected EntityHanging(EntityTypes<? extends EntityHanging> entitytypes, World world) {
+         super(entitytypes, world);
+         this.direction = EnumDirection.SOUTH;
++        this.e = this.getId() % this.world.spigotConfig.hangingTickFrequency; // Paper
```

Comment:
We can make this diff smaller & avoid an obfhelper by replacing the field with this:
`    private int e; { this.e = this.getId() % this.world.spigotConfig.hangingTickFrequency; }`

Reply:
Done